File: server/tests/fixtures/alpha_rule_triggers/csharp/arch_global_state_usage.cs
// Should trigger: arch.global_state_usage
public class GlobalState {
    public static int Counter = 0;
    
    public void DoSomething() {
        // Access global state via Environment singleton - triggers arch.global_state_usage
        string user = Environment.UserName;
        string machine = Environment.MachineName;
    }
}

File: server/tests/fixtures/alpha_rule_triggers/java/arch_global_state_usage.java
// Should trigger: arch.global_state_usage
public class GlobalState {
    public static int counter = 0;
    
    public void doSomething() {
        // Access global state via System singleton - triggers arch.global_state_usage
        String prop = System.getProperty("user.home");
        Runtime rt = Runtime.getRuntime();
    }
}
